packages/bigtable/src/table.js

```diff
@@ -861,8 +867,11 @@ Table.prototype.mutate = function(entries, callback) {
     })
   ]);
 
-  if (isStreamMode) {
-    return stream;
+  if (!isCallbackMode) {
+    ['error', 'finish'].forEach(function(event) {
```

Comment:
Can you rename `concat(function(entries) {}` to `concat(function(insertErrors) {}`?